### ISSUE TITLE
Age filter breaks in TransfersSearchResult page when you have an already sold/unsold player

### ISSUE THREAD

**Issue by minj** [minj/foxtrick#842]
Original [issue 842](https://code.google.com/p/foxtrick/issues/detail?id=842) created by [lizardopoli](mailto:lizardopoli@gmail.com) on 2011-10-31T15:54:15.000Z:
- What steps will reproduce the problem?

Make a common search, in which you will find for sure an unsold or already sold player
- What is the expected behaviour? What is the actual behaviour instead? You could provide a screenshot if necessary.

I expect the age filter working, but it doesn't work.
- What is the version of the FoxTrick, your browser and operating system?

Foxtrick 0.8, Firefox 7.0.1, Windows 7 x64 
- Copy FoxTrick debug output on the page here (Enable debug output at main preferences tab of FoxTrick)

FoxTrick Log

Version 0.8, Gecko Firefox platform, it locale, standard layout, LTR direction
chrome://foxtrick/content/shortcuts-and-tweaks/transfer-search-result-filters.js (21): TypeError: player.age is undefined
Stack trace: ([object Object],null,4)@chrome://foxtrick/content/shortcuts-and-tweaks/transfer-search-result-filters.js:21
([object Array])@chrome://foxtrick/content/shortcuts-and-tweaks/transfer-search-result-filters.js:212
([object Array])@chrome://foxtrick/content/shortcuts-and-tweaks/transfer-search-result-filters.js:45
(&quot;transfer-search-result-filters&quot;,(function (n) {try {if (n === undefined) {Foxtrick.sessionSet(&quot;transfer-search-result-filters&quot;, FoxtrickTransferSearchResultFilters.FILTER_VAL);callback(FoxtrickTransferSearchResultFilters.FILTER_VAL);} else {callback(n);}} catch (e) {Foxtrick.log(e);}}))@chrome://foxtrick/content/util/session-store.js:24
((function (filters) {var playerList = Foxtrick.Pages.TransferSearchResults.getPlayerList(doc);var playerInfos = doc.getElementsByClassName(&quot;transferPlayerInfo&quot;);playerInfos = Foxtrick.map(function (n) {return n;}, playerInfos);for (var i = 0; i &lt; playerInfos.length; ++i) {var player = playerList[i];var hide = false;for (var j = 0; j &lt; filters.l
run time: 109 ms | /World/Transfers/TransfersSearchResult.aspx
- Please provide any additional information below.

It's a problem I had with psicotsi too. That happens because there is a &quot;break&quot; in a loop instead of a continue. 

In other words: &quot;for (players in page)... if (player_table is not found) break&quot; that &quot;break&quot; should be &quot;continue&quot; instead

Best Regards
